frontend/package.json

```diff
@@ -5,20 +5,34 @@
   "main": "index.js",
   "scripts": {
     "start": "react-scripts start",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "webpack --mode production",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage --colors"
+  },
+  "jest": {
+    "snapshotSerializers": [
+      "enzyme-to-json/serializer"
+    ]
   },
   "author": "Vitalii Posidailo",
   "license": "ISC",
+  "babel": {
+    "presets": [
+      "@babel/preset-env",
+      "@babel/preset-react"
+    ]
+  },
```

Comment:
You probably don't need to specify babel both in `package.json` and `.babelrc` files. One place is enough.

Reply:
Done. Removed from package.json

Reply:
Looks like you haven't pushed this change.

Reply:
Pushed all latest changes into the branch.